<launch>
  <node pkg="rover_platform" name="projMapToImage" type="projMapToImage" output="screen">
  </node>

  <node pkg="rover_platform" name="findFrontier" type="findFrontier" output="screen">
  </node>
</launch>
